app/views/users/logix/search.html.erb

```diff
@@ -12,11 +12,13 @@
 <% else %>
   <div class="container search-container">
     <% @results.map { |user| ProfileDecorator.new(user) }.each do |profile| %>
-      <div class="row center-row search-user-container">
+      <div class="row center-row search-user-container border mb-4">
```

Comment:
Purpose of "border"? Also can we move mb-4 to search-user-container css if its not conflicting with other styles

Reply:
Also, I think you will be required to replace search-user-container with search-card-container to add a border to individual card. See if we are using search-user-container anywhere else or remove it

Reply:
search-user-container is not used at anywhere else so removing class from erb as well as stylesheet
I thought of using search-card-container but it is making each card as big as cards in project search
smaller usercards looked more tiny to me!!
 so i settled up with adding border manually
changed search-user-container with search-card-container

Reply:
![2020-10-20_10-22](https://user-images.githubusercontent.com/55848322/96541795-61461080-12be-11eb-967e-75dfcd7ee5bb.png)

Reply:
Define something like search-user-card-container and use that. We require cards with less width in this page as earlier. That class would account for everything from width to border

Reply:
![Screenshot from 2020-10-22 12-37-43](https://user-images.githubusercontent.com/55848322/96837109-858f2200-1463-11eb-91bf-aa5d13c2f33a.png)

Reply:
Everything else looks great; @pavanjoshi914 can we just decrease the size of the user image a bit and move all content in each card a bit below, adding more padding to the top.

Reply:
Ohk making following changes thanks!!